sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/PredicateSuite.scala

```diff
@@ -523,34 +523,6 @@ class PredicateSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   val row0 = create_row(null)
```

Comment:
Can you move this variable into the test `test("isunknown and isnotunknown")`?

Reply:
OK